demo/addons/fmod/Fmod.gd

```diff
@@ -151,6 +156,42 @@ func shutdown() -> void:
 	else:
 		print("Fmod not started, can't shutdown'")
 
+func load_configuration(file_path: String, loaded_from_ready: bool = false) -> void:
+	if started:
+		shutdown()
+	
+	var config = ConfigFile.new()
+	var err = config.load(file_path)
+	
+	if loaded_from_ready:
+		if !config.get_value("Init", "LoadFMODOnStart"):
+			return
+	
+	#Set software format and set DSP buffer
+	set_software_format(config.get_value("Init", "SampleRate"), config.get_value("Init", "SpeakerMode"), 0)
+	set_dsp_buffer_size(config.get_value("Init", "DSPBufferSize"), 4)
+		
+	#Set mode for studio to init in (Normal or Live Update)
+	var studio_init_mode = FMOD_STUDIO_INIT_LIVEUPDATE
+	if !config.get_value("Init", "LiveUpdate"):
+		studio_init_mode = FMOD_STUDIO_INIT_NORMAL
+		
+	#Initalise FMOD
+	init(config.get_value("Init", "NumChannels"), studio_init_mode, FMOD_INIT_NORMAL)
+	set_sound_3D_settings(config.get_value("3D", "DoplerScale"), config.get_value("3D", "DistanceFactor"), config.get_value("3D", "RolloffScale"))
+	set_listener_number(config.get_value("Init", "NumListeners"))
+	
+	#Load FMOD Autoload Banks
+	var banks = config.get_value("Banks", "Autoload").split("\n", false)
+	for bank in banks:
```

Comment:
FMOD requires an order for the banks to be loaded.
First the master.strings.bank
second the master.bank
third everything else.
In the demo project we are just lucky with the alphabetical order of the files so it's the right order but you have to enforce that in the code.